tasks/release.rb

```diff
@@ -44,13 +44,21 @@
       raise "Could not insert PRE in #{file}" unless $1
 
       File.open(file, 'w') { |f| f.write ruby }
+
+      if framework == "actioncable"
+        Dir.chdir("#{root}/actioncable") do
+          version = version.gsub(/\./).with_index { |s, i| i === 2 ? '-' : s }
```

Comment:
Extract an `#npmify_version` or similar that reliably transforms a RubyGems version to an NPM version. Checking for the 3rd `.` _seems_ brittle, but will prob work fine for our versions.